xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -61,32 +63,42 @@ CDVDSubtitlesLibass::CDVDSubtitlesLibass()
 
   ass_set_message_cb(m_library, libass_log, this);
 
+  CLog::Log(LOGINFO, "CDVDSubtitlesLibass: Initializing ASS Renderer");
+
+  m_renderer = ass_renderer_init(m_library);
+
+  if (!m_renderer)
```

Comment:
this should probably `throw std::runtime_error` because if this fails you may be in a state you can't recover from. In the constructor you should consider RAII techniques where you allocate what you need, then later in another method (after the constructor succeeds) consider using a `Configure` method or similar.